ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -64,4 +64,7 @@ conformance_test(
         5021,
     ],
     server = ":canton-test-runner-with-dependencies",
+    test_tool_args = [
+        "--exclude=PartyAllocationIT", # TODO Review: pls. confirm Canton implements validation separately
```

Comment:
@oliverse-da You are probably very well qualified to answer this.

Reply:
@oliverse-da Actually I could also modify the test to also cover Canton's current message (or add a Canton version of it) and include it if you think it makes sense.